Makefile

```diff
@@ -55,20 +54,6 @@ API_SPECS := \
     components/renku_data_services/data_connectors/apispec.py \
     components/renku_data_services/search/apispec.py
 
-components/renku_data_services/crc/apispec.py: components/renku_data_services/crc/api.spec.yaml
-components/renku_data_services/storage/apispec.py: components/renku_data_services/storage/api.spec.yaml
-components/renku_data_services/users/apispec.py: components/renku_data_services/users/api.spec.yaml
-components/renku_data_services/project/apispec.py: components/renku_data_services/project/api.spec.yaml
-components/renku_data_services/session/apispec.py: components/renku_data_services/session/api.spec.yaml
-components/renku_data_services/namespace/apispec.py: components/renku_data_services/namespace/api.spec.yaml
-components/renku_data_services/secrets/apispec.py: components/renku_data_services/secrets/api.spec.yaml
-components/renku_data_services/connected_services/apispec.py: components/renku_data_services/connected_services/api.spec.yaml
-components/renku_data_services/repositories/apispec.py: components/renku_data_services/repositories/api.spec.yaml
-components/renku_data_services/notebooks/apispec.py: components/renku_data_services/notebooks/api.spec.yaml
-components/renku_data_services/platform/apispec.py: components/renku_data_services/platform/api.spec.yaml
-components/renku_data_services/data_connectors/apispec.py: components/renku_data_services/data_connectors/api.spec.yaml
-components/renku_data_services/search/apispec.py: components/renku_data_services/search/api.spec.yaml
```

Comment:
We can just remove this and not do the file renaming and everything will still work.

Reply:
done